packages/dashboard/package.json

```diff
@@ -39,42 +37,66 @@
     "@mantine/hooks": "^5.0.0",
     "@mantine/notifications": "^5.0.0",
     "@mantine/prism": "^5.0.0",
+    "@truffle/codec": "^0.14.8",
+    "@truffle/compile-common": "^0.9.0",
```

Comment:
FWIW, I think it's totally reasonable to take a dep on `compile-common` here.

Reply:
Yea the types are important for type-checking and schema definition. Your point about `WorkflowCompileResultMessage` being too narrow still perfectly stands though.